### PR TITLE
Bdu 83 revamp big dipper landing page

### LINKED ISSUE #8: UI: Home/ Carousel

### LINKED ISSUE #9: UI: Home/ Supported networks

### LINKED ISSUE #10: UI: All networks/ Title, text, and search bar
- title
- text
- search bar

### LINKED ISSUE #11: UI: All networks/ Networks + Pull over different nets
截圖 2022-05-23 下午11 44 36

### LINKED ISSUE #12: UI: All networks/ Tabs + Up arrow

### LINKED ISSUE #25: UI: 404 page
- check design

### LINKED ISSUE #26: Fix background gradient

### LINKED ISSUE #36: UI: Implement BD landing page Header
Revised the menu bar using the following structure
```
Home
All Networks
Helps & Support
├ FAQ
├ Donation
├ Feedback
About
Docs
```

Add new big dipper logo

### PR DESCRIPTION
**Pull request by teamchong**
Closes #36 
Closes #26 
Closes #12 
Closes #11 
Closes #10 
Closes #9 
Closes #8 
Closes #25 

## Checklist
- [X] Ran Linting
- [X] Targeted PR against correct branch.
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [X] Wrote unit tests.
- [X] Added an entry to the `CHANGELOG.md` file.
- [X] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-node@v3
```
Comment (teamchong) [inline .github/workflows/test.yml:9]: ff

**Review by yayay927 — CHANGES_REQUESTED** [review]
Hi Steven, since I noticed there are some parts different from design, I'm going to leave all the comments about UI in this pr.

1. Current home page background color gradient (image 2) doesn't fit design wireframe (image 1)
截圖 2022-08-24 上午1 56 10
(image 1)
截圖 2022-08-24 上午1 59 00
(image 2)

2. Supported networks page -- row number in your staging (8) doesn't fit design wireframe (5)
截圖 2022-08-24 上午2 01 52

3. Circled arrow under supported networks component doesn't fit design wireframe, which should be a round circle instead of an oval
截圖 2022-08-24 上午2 02 34

4. There are colors for different selected in design wireframe for menu items
截圖 2022-08-24 上午2 06 33

5. Font for menu is different from design wireframe (as image above)

**Comment by teamchong** [comment]
> Hi Steven, since I noticed there are some parts different from design, I'm going to leave all the comments about UI in this pr.
> 
> 1. Current home page background color gradient (image 2) doesn't fit design wireframe (image 1)
>    (image 1)
> 
> 截圖 2022-08-24 上午1 56 10
> 
> (image 2) 截圖 2022-08-24 上午1 59 00
> 
> 2. Supported networks page -- row number in your staging (8) doesn't fit design wireframe (5)
> 
> 截圖 2022-08-24 上午2 01 52
> 
> 3. Circled arrow under supported networks component doesn't fit design wireframe, which should be a round circle instead of an oval
> 
> 截圖 2022-08-24 上午2 02 34
> 
> 4. There are colors for different selected in design wireframe for menu items
> 
> 截圖 2022-08-24 上午2 06 33
> 
> 5. Font for menu is different from design wireframe (as image above)

done

**Comment by teamchong** [comment]
@yayay927 please help to follow up

**Review by yayay927 — CHANGES_REQUESTED** [review]
截圖 2022-09-23 下午4 17 32
Need more margin top above header. Right now the logo and buttons are too close to the top.

截圖 2022-09-23 下午4 20 22
The hover animation of chosen button brights up in the last second too suddenly. Unlike the natural hover animation of white unchosen items.

**Review by yayay927 — CHANGES_REQUESTED** [review]
截圖 2022-09-23 下午4 24 32
(3) Right now the white title is displayed in different height in different carousel images. Which creates a popping/moving visual effect for users and it's not very comfortable to stare at, especially when wanting to read the titles and descriptions carefully. Is it possible to make them in same horizontal line?
(4) The same with above, the main images are in different sizes, which also creates the kind of popping visual effect. Is it possible to adjust them into same visual size on page?

**Comment by teamchong** [comment]
> 截圖 2022-09-23 下午4 17 32
> 
> (1) Need more margin top above header. Right now the logo and buttons are too close to the page top.
> 截圖 2022-09-23 下午4 20 22
I had reduce the logo size
> 
> (2) The hover animation of chosen button brights up in the last second too suddenly. Unlike the natural hover animation of white unchosen items.

removed hover effect for active menu item

**Pull request merged**